### ISSUE TITLE
False positive for SlevomatCodingStandard.ControlStructures.RequireSingleLineCondition.RequiredSingleLineCondition

### ISSUE THREAD

**Issue by spaceemotion** [slevomat/coding-standard#1119]
The following code throws 3 errors:

```php
if ( // 28
    in_array($brokerId->type_id, $types) // 29
    && ($status === null
        || in_array($brokerId->status, $status)) // 31
) {
    $brokerIds[] = $brokerId;
}
```
```yaml
  28: [ERROR] Condition of "if" should be placed on a single line. (php_codesniffer)
              SlevomatCodingStandard.ControlStructures.RequireSingleLineCondition.RequiredSingleLineCondition

  29: [ERROR] Strict parameter missing in in_array() call. (php_codesniffer)
              SlevomatCodingStandard.Functions.StrictCall.StrictParameterMissing

  31: [ERROR] Strict parameter missing in in_array() call. (php_codesniffer)
              SlevomatCodingStandard.Functions.StrictCall.StrictParameterMissing
```

But if the strict parameters are getting added, all errors go away:

```php
if (
    in_array($brokerId->type_id, $types, true)
    && ($status === null
        || in_array($brokerId->status, $status, true))
) {
    $brokerIds[] = $brokerId;
}
```

---

Interestingly enough, trying to "fix" the if check (and not the strict calls), throws linting errors in PHP_CodeSniffer itself:

```php
if (
    in_array($brokerId->type_id, $types, true)
    && ( // 30
        $status === null
        || in_array($brokerId->status, $status, true)
    ) // 33
) {
    $brokerIds[] = $brokerId;
}
```
```yaml
  30: [ERROR] Expected 0 space after open parenthesis; newline found (php_codesniffer)
              Generic.WhiteSpace.ArbitraryParenthesesSpacing.SpaceAfterOpen

  33: [ERROR] Expected 0 space before close parenthesis; newline found (php_codesniffer)
              Generic.WhiteSpace.ArbitraryParenthesesSpacing.SpaceBeforeClose
```

**Comment by kukulich** [comment]
When you add `true` twice the `if` condition is longer (12 letters are added) so the single line `if` will probably not match your line limit so the error is not reported again :)
